### ISSUE TITLE
Rultor Reloads GPG Keys on every Build when Decrypting, Failing to Communicate with the Keyserver Fails Builds Needlessly

### ISSUE THREAD

**Issue by original-brownbear** [yegor256/rultor#1015]
This is coming from https://github.com/yegor256/rultor/issues/1003:

Currently we have this code in Rultor:
```
    /**
     * Decrypt instructions.
     * @return Instructions
     * @throws IOException If fails
     */
    public Iterable<String> commands() throws IOException {
        final Collection<XML> assets =
            this.profile.read().nodes("/p/entry[@key='decrypt']/entry");
        final Collection<String> commands = new LinkedList<String>();
        if (!assets.isEmpty()) {
            commands.add("gpgconf --reload gpg-agent");
            commands.add(
                Joiner.on(' ').join(
                    "gpg --keyserver hkp://pool.sks-keyservers.net",
                    this.proxy,
                    "--verbose --recv-keys 9AF0FA4C"
                )
            );
            commands.add("gpg --version");
        }
```

Part of the issues reported in https://github.com/yegor256/rultor/issues/1003 are a result of the kall to the keyserver sporadically failing. Failing to contact the keyserver needs to be handled in a more reasonable and stable manner. Rultor needs to reload the key from a different server in case the given one fails, this should never be something that breaks a build given the wide availability of key servers.

**Comment by original-brownbear** [comment]
@alex-palevsky this is urgent.

**Comment by original-brownbear** [comment]
@alex-palevsky this is a bug.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is a bug.

@original-brownbear tag bug added to this issue

**Comment by alex-palevsky** [comment]
@original-brownbear I set milestone here to `2.0`, let me know if it is wrong

**Comment by alex-palevsky** [comment]
@original-brownbear thanks a lot for reporting, **30 mins** added to your acc, pmt ID `AP-7UH92031H0578253S`

**Comment by original-brownbear** [comment]
@alex-palevsky this depends on #1055

**Comment by alex-palevsky** [comment]
> @alex-palevsky this depends on #1055

@original-brownbear OK, let's wait for #1055

**Comment by alex-palevsky** [comment]
@original-brownbear I think we can continue, impediment #1055 is closed

**Comment by original-brownbear** [comment]
@alex-palevsky this is postponed.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear sure, thanks, I added "postponed" label to it

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear right, I will find someone else, no problem
